Makefile

```diff
@@ -131,3 +132,9 @@ docker_push: docker_tag docker_auth ## push to docker hub
 	docker push 'openedx/edx-platform:latest-devstack'
 	docker push "openedx/edx-platform:${GITHUB_SHA}-devstack"
 
+upgrade-importlinter:  # Temporary convenience rule.
+	pip-compile -v --no-emit-trusted-host --no-index --rebuild --upgrade -o requirements/edx/importlinter.txt requirements/edx/importlinter.in
+
+lint-imports:
+	pip install -r requirements/edx/importlinter.txt
+	lint-imports --config importlinter.cfg
```

Comment:
we could instead put contracts in `.importlinter` and obviate the `--config` argument here, but I liked the GitIHub syntax highlighting that the `.cfg` file gives us.